doc/man/scons.xml

```diff
@@ -1120,24 +1120,48 @@ the mechanisms in the specified order.</para>
   </listitem>
   </varlistentry>
 
+  <varlistentry id="opt--experimental">
+    <term>
+      <option>--experimental=<replaceable>all</replaceable></option>,
+      <option>--experimental=<replaceable>none</replaceable></option>
+    </term>
+    <listitem>
+      <para>Enable experimental features and/or tools. <emphasis role="strong">No Support offered for any features or tools enabled by this flag</emphasis></para>
+      <para>The default setting is none.</para>
+    </listitem>
+  </varlistentry>
+
+
   <varlistentry id="opt-f">
-  <term>
-    <option>-f <replaceable>file</replaceable></option>,
-    <option>--file=<replaceable>file</replaceable></option>,
-    <option>--makefile=<replaceable>file</replaceable></option>,
-    <option>--sconstruct=<replaceable>file</replaceable></option>
-  </term>
-  <listitem>
-<para>Use
-<replaceable>file</replaceable>
-as the initial SConscript file.
-Multiple
-<option>-f</option>
-options may be specified,
-in which case
-&scons;
-will read all of the specified files.</para>
-  </listitem>
+    <term>
+      <option>-f
+        <replaceable>file</replaceable>
+      </option>
+      ,
+      <option>--file=
+        <replaceable>file</replaceable>
+      </option>
+      ,
+      <option>--makefile=
+        <replaceable>file</replaceable>
+      </option>
+      ,
+      <option>--sconstruct=
+        <replaceable>file</replaceable>
+      </option>
+    </term>
+    <listitem>
+      <para>Use
+        <replaceable>file</replaceable>
+        as the initial SConscript file.
+        Multiple
+        <option>-f</option>
+        options may be specified,
+        in which case
+        &scons;
+        will read all of the specified files.
```

Comment:
this seems sort of odd (not a comment on the patch)... what does it mean to have multiple SConstruct-type files specified by -f? Which one is then "the" initial file for purposes such as determining the topdir? Plus the logic in `Main.py` seems funky here:
```
    scripts = []
    if options.file:
        scripts.extend(options.file)
    if not scripts:
        sfile = _SConstruct_exists(repositories=options.repository,
                                   filelist=options.file)
        if sfile:
            scripts.append(sfile)
```
so if you have one or more argument-specified "SConstruct" files, you don't check if any of them exist, because `scripts` will then be non-empty.

Reply:
Seems likely, but that not new code for this PR.

Reply:
Okay, so I need to find someplace else for this comment - suggestions?